javascript/index.js

```diff
@@ -14,15 +14,24 @@ let milUni = document.getElementById('milUni');
 let splits = document.getElementById('splits');
 
 function printTime() {
-  // ... your code goes here
+  getMinutes()
+  getSeconds()
 }
 
 function printMinutes() {
-  // ... your code goes here
+  let min = chronometer.twoDigitsNumber(chronometer.getMinutes());
+  let minSplit = min.split('');
```

Comment:
no need to split the string into an array to use bracket notation, you can also use bracket notation on strings :)